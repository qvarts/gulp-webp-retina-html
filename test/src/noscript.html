<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Webp images</title>
</head>
<body>
<img src="../img/test.jpg" alt="test" />
<img data-src="../img/test.png" alt="test" />
<img data-src="../img/test.png" src="../img/fallback.gif" alt="test" />
</body>
</html>