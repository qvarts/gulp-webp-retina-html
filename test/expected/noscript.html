<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Webp images</title>
</head>
<body>
<picture>
<source srcset="../img/test.webp" type="image/webp">
<img src="../img/test.jpg" alt="test" />
</picture>
<noscript><img src="../img/test.png" alt="test" /></noscript>
<picture>
<source data-srcset="../img/test.webp" type="image/webp">
<img data-src="../img/test.png" alt="test" />
</picture>
<noscript><img src="../img/test.png" alt="test" /></noscript>
<picture>
<source data-srcset="../img/test.webp" srcset="../img/fallback.gif" type="image/webp">
<img data-src="../img/test.png" src="../img/fallback.gif" alt="test" />
</picture>
</body>
</html>